DOD-ISD-Verbs/dodisdverbs.jsonld

```diff
@@ -0,0 +1,4729 @@
+			}
+		},
+		{
+			"@id": "https://w3id.org/xapi/dod-isd/verbs/disposed",
+			"inScheme": "https://w3id.org/xapi/dod-isd",
+			"@type": "Verb",
+			"prefLabel": {
+				"en": "disposed"
+			},
+			"definition": {
+				"en": "Eliminating something or someone by throwing away, removing, or destroying."
+			}
+		},
+		{
+			"@id": "https://w3id.org/xapi/dod-isd/verbs/disseminated",
+			"inScheme": "https://w3id.org/xapi/dod-isd",
+			"@type": "Verb",
+			"prefLabel": {
+				"en": "disseminated"
+			},
+			"definition": {
+				"en": "Spread or disperse something, especially information widely."
+			}
+		},
+		{
+			"@id": "https://w3id.org/xapi/dod-isd/verbs/drove",
+			"inScheme": "https://w3id.org/xapi/dod-isd",
+			"@type": "Verb",
+			"prefLabel": {
+				"en": "drove"
+			},
+			"definition": {
+				"en": "Propel or carry along by force in a specified direction; operate and control the direction and speed of a vehicle."
+			}
+		},
+		{
+			"@id": "https://w3id.org/xapi/dod-isd/verbs/egressed",
+			"inScheme": "https://w3id.org/xapi/dod-isd",
+			"@type": "Verb",
+			"prefLabel": {
+				"en": "egressed"
+			},
+			"definition": {
+				"en": "Go out of or leave a place."
+			}
+		},
+		{
+			"@id": "https://w3id.org/xapi/dod-isd/verbs/elevated",
+			"inScheme": "https://w3id.org/xapi/dod-isd",
+			"@type": "Verb",
+			"prefLabel": {
+				"en": "elevated"
+			},
+			"definition": {
+				"en": "Raise or lift something up to a higher position."
+			}
+		},
+		{
+			"@id": "https://w3id.org/xapi/dod-isd/verbs/emplaced",
+			"inScheme": "https://w3id.org/xapi/dod-isd",
+			"@type": "Verb",
+			"prefLabel": {
+				"en": "emplaced"
+			},
+			"definition": {
+				"en": "Put in place or position."
+			}
+		},
+		{
+			"@id": "https://w3id.org/xapi/dod-isd/verbs/employed",
+			"inScheme": "https://w3id.org/xapi/dod-isd",
+			"@type": "Verb",
+			"prefLabel": {
+				"en": "employed"
+			},
+			"definition": {
+				"en": "Make use of something or someone; provide work to someone for payment."
+			}
+		},
+		{
+			"@id": "https://w3id.org/xapi/dod-isd/verbs/engaged",
+			"inScheme": "https://w3id.org/xapi/dod-isd",
+			"@type": "Verb",
+			"prefLabel": {
+				"en": "engaged"
+			},
+			"definition": {
+				"en": "Occupy, attract, or involve someone; enter into conflict or combat with an enemy."
+			}
+		},
+		{
+			"@id": "https://w3id.org/xapi/dod-isd/verbs/energized",
+			"inScheme": "https://w3id.org/xapi/dod-isd",
+			"@type": "Verb",
+			"prefLabel": {
+				"en": "energized"
+			},
+			"definition": {
+				"en": "Give energy, vitality and/or enthusiasm to someone or something."
+			}
+		},
+		{
+			"@id": "https://w3id.org/xapi/dod-isd/verbs/entered",
+			"inScheme": "https://w3id.org/xapi/dod-isd",
+			"@type": "Verb",
+			"prefLabel": {
+				"en": "entered"
+			},
+			"definition": {
+				"en": "Come or go into a place."
+			}
+		},
+		{
+			"@id": "https://w3id.org/xapi/dod-isd/verbs/established",
```

Comment:
This seems underspecified. Also, while some of the listed values are exclusive, others don't seem like they would be. Is it really only possible for one of them to relate to a given statement?

Reply:
In the case of interactivity level - the answer is no. Some verbs can be experienced with varying levels of interactivity according to the DoD documentation. However, the SKA data is very specific to each verb. I confess, some of the verbs were perplexing - but considering their orientation to military activities they made sense. I anticipate establishing a CoP to help the ISD community come together better on their use within training objectives. As it stands, some verbs are used more frequently while others are neglected - we have seen this on many programs where the established training needs analysis got lazy and used very little variety in verbs. Usually that IMI was level 1 or 2 interactivity - but now we are moving training into much higher levels of interactivity and tracking more events from simulators combining that information with more traditional academic training that programs which do not reach back to the original intent behind these DoD verbs could be considered ineffective in that they do not describe what the student is actually supposed to learn.